pkg/cmd/cli/cli.go

```diff
@@ -64,8 +50,8 @@ func NewCommandCLI(name, fullName string, in io.Reader, out, errout io.Writer) *
 	}
 
 	templates.ActsAsRootCommand(cmds, filters, groups...).
-		ExposeFlags(getCmd, "server", "client-certificate",
-			"client-key", "certificate-authority", "insecure-skip-tls-verify", "token")
+		ExposeFlags(groups[0].Commands[0], "server", "client-certificate",
```

Comment:
I'm mostly unfamiliar, but is there any chance that the magic number 0's ever become not-so-magic?